CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Release notes
 This page documents production updates to Cloud Tools for IntelliJ. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## 18.6.1
+
+### Fixed
+  - Fixed errors in the managed Cloud SDK cause by downgrading the version. [666](https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/666)
```

Comment:
caused?

Reply:
thanks. fixed

Reply:
I'd be more specific about what behavior we specifically fixed.  We really want to call out to users that experienced these installation failures that they're fixed.